storage/mysql/log_storage.go

```diff
@@ -51,8 +51,9 @@ const (
 			VALUES(?,0,?,?,?)`
 	insertSequencedLeafSQL = `INSERT INTO SequencedLeafData(TreeId,LeafIdentityHash,MerkleLeafHash,SequenceNumber)
 			VALUES(?,?,?,?)`
-	selectSequencedLeafCountSQL  = "SELECT COUNT(*) FROM SequencedLeafData WHERE TreeId=?"
-	selectLatestSignedLogRootSQL = `SELECT TreeHeadTimestamp,TreeSize,RootHash,TreeRevision,RootSignature
+	selectSequencedLeafCountSQL   = "SELECT COUNT(*) FROM SequencedLeafData WHERE TreeId=?"
```

Comment:
if there are 20B entries in the queue, then I'd want to know - even if that data was 20mins out of date.
We can build quicker proxy metrics (e.g. `Rate(Sum(ctfe.num_added)) - Rate(Sum(signer.sequenced)))`) too, but ground-truth is the number of rows in the table, and it'd be nice to have that on a graph.

If this murders the DBs then we can revisit (this is really what I meant by "alright for now").